### PR TITLE
cmake: install missing *.hpp

### PR DESCRIPTION
**Pull request by kenhys**
groonga.hpp and arrow.hpp are missing.

### CODE REVIEW

File: include/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,11 @@
 
 install(FILES groonga.h DESTINATION "${GRN_INCLUDE_DIR}")
```
Comment (komainu8) [inline include/CMakeLists.txt]: Do we needless install "groonga.hpp"?

Comment (kenhys) [reply include/CMakeLists.txt]: Oh, sure.

**Comment by kenhys** [comment]
@komainu8 Thanks!

I've fixed it.

Also, I've found the problem that groonga.h is installed to a wrong path, so I've also fixed it.

**Comment by kenhys** [comment]
Hmm, not completed yet... it need more fixes.

**Comment by kenhys** [comment]
https://github.com/groonga/groonga/commit/61de0f9ded701eff88a31d1a25d29423fc603d2a is not needed.
@komainu8 now, ready for review again.

**Comment by kenhys** [comment]
Thanks!

**Pull request merged**